src/billing/reducers/index.ts

```diff
@@ -93,7 +93,7 @@ export const setInvoicesStatus = (status: RemoteDataState) =>
 
 export const setPaymentMethods = (
   paymentMethods: PaymentMethod[],
-  creditCards: CreditCardParams,
+  creditCards: ZuoraParams,
```

Comment:
CreditCards and ZuoraParams are two different things based on the new yml contract. We will still want creditCards here

Reply:
I don't see a route which uses ZuoraParams in the yml file.

Reply:
Seems like ZuoraParams is now CreditCardParams.